<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Specific</class>
 <widget class="QWidget" name="Specific">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1225</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="emotionlabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>您今天的心情是：</string>
   </property>
  </widget>
  <widget class="QLabel" name="emotion">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="song">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="songlabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>您可以尝试听听这首歌：</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>441</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>您今天的随机日程生成如下</string>
   </property>
   <widget class="QLabel" name="label8">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>20</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>08：00-10：00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label10">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>40</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>10：00-12：00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label14">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>60</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>14：00-16：00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label16">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>80</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>16：00-18：00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label18">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>100</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>18：00-20：00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label20">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>120</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>20：00-22：00</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="Diary">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>90</y>
     <width>301</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="OK_Button">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>430</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>选择</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Save_Button">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>460</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="Calendar">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>150</y>
     <width>311</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>550</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="DateLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>20</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>每日小记</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Pause_Continue">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>暂停/继续</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>20</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>生成年度总结</string>
   </property>
  </widget>
  <widget class="QLabel" name="schedule8">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>370</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>无</string>
   </property>
  </widget>
  <widget class="QLabel" name="schedule18">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>450</y>
     <width>350</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>无</string>
   </property>
  </widget>
  <widget class="QLabel" name="schedule20">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>470</y>
     <width>350</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>无</string>
   </property>
  </widget>
  <widget class="QPushButton" name="volume_down">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>音量 -</string>
   </property>
  </widget>
  <widget class="QPushButton" name="volume_up">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>音量 +</string>
   </property>
  </widget>
  <widget class="QLabel" name="schedule10">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>390</y>
     <width>350</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>无</string>
   </property>
  </widget>
  <widget class="QLabel" name="schedule14">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>410</y>
     <width>350</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>无</string>
   </property>
  </widget>
  <widget class="QLabel" name="schedule16">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>430</y>
     <width>350</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>无</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
